.changeset/violet-shirts-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ai-sdk/fireworks': patch
+---
+
+feat (provider/fireworks): Pass generateImage sample count to backend.
```

Comment:
feat (provider/fireworks): Send generateImage n parameter to Fireworks API as 'samples'.